publisher/internal/analytics/ethereum/operations.go

```diff
@@ -344,7 +340,7 @@ func (op OperationBase) lookupPrice(address string) (repository.TokenPrice, erro
 // ----------------------------------------------------------------------
 // --------------- Position methods
 
-func (pos *Position) calculateInterval() {
+func (pos *Position) calculate() {
 
 	lowerRatio := convertTickToRatio(pos.LowerTick, pos.Token0.Decimals, pos.Token1.Decimals)
```

Comment:
Can `convertTickToRatio` be a `Position` method and called `getTokenRatio`?

Reply:
I would prefer to keep it independent as it converts provided tick to ratio, based on decimals of two tokens used, no other position-specific information is required. This function might be used in different scenarios.